configs/11.0/packages/gdb/sources

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# GDB source package and build info
+# =================================
+#
+ATSRC_PACKAGE_NAME="GDB (GNU Debugger)"
+ATSRC_PACKAGE_VER=7.11.1
```

Comment:
I think AT 10.0 and next already have a newer GDB.
I'd just symlink this file to AT 10.0.